components/pkg-export-docker/defaults/Dockerfile_win.hbs

```diff
@@ -5,7 +5,7 @@ EXPOSE 9631 {{exposes}}
 RUN SET HAB_FEAT_OFFLINE_INSTALL=ON && \
     {{~ #if environment}}
     {{~ #each environment}}
-        SET {{@key}}={{{this}}} && \
+        SET {{@key}}={{{this}}}&& \
```

Comment:
Would
```suggestion
        SET {{@key}}="{{{this}}}" && \
```
work as well? It's strange to have to space before the `&&`, so if this handles the issue, quoting the RHS might be a good convention to adopt.

Reply:
no that would include the quotes in the value

Reply:
Oh, wild. TIL.